lib/src/main/java/com/springboot/webpush/common/api/OnActionClick.java

```diff
@@ -1,32 +1,42 @@
 package com.springboot.webpush.common.api;
 
+/**
+ * The {@link OnActionClick} class is used to control how push messages are to behave. Valid operations are:<br>
+ * openWindow Opens a new tab at the specified URL.<br>
+ * focusLastFocusedOrOpen Focuses the last focused client. If there is no client open, then it opens a new tab at the
+ * specified URL. (url is optional)<br>
+ * navigateLastFocusedOrOpen Focuses the last focused client and navigates it to the specified URL. If there is no
+ * client open, then it opens a new tab at the specified URL.<br>
+ * sendRequest Send a simple GET request to the specified URL.<br>
+ *
+ */
 public class OnActionClick {
-    /* valid operations are:
-         openWindow	Opens a new tab at the specified URL.
-         focusLastFocusedOrOpen	Focuses the last focused client. If there is no client open, then it opens a new tab at the specified URL. (url is optional)
-         navigateLastFocusedOrOpen	Focuses the last focused client and navigates it to the specified URL. If there is no client open, then it opens a new tab at the specified URL.
-         sendRequest	Send a simple GET request to the specified URL.
-    */
     private final String operation;
```

Comment:
Maybe this wants to be an enum, or have some sanity checking?

Reply:
Looks like you created one, but needs to be used here

Reply:
Resolved.